src/main/java/com/mraof/minestuck/data/AspectTreeBlocksData.java

```diff
@@ -21,845 +21,841 @@ public final class AspectTreeBlocksData
 {
 	public static void addEnUsTranslations(MinestuckLanguageProvider provider)
 	{
-		provider.addBlock(BLOOD_ASPECT_LOG, "Blood Log");
-		provider.addBlock(BLOOD_ASPECT_WOOD, "Blood Wood");
-		provider.addBlock(BLOOD_ASPECT_STRIPPED_LOG, "Stripped Blood Log");
-		provider.addBlock(BLOOD_ASPECT_STRIPPED_WOOD, "Stripped Blood Wood");
-		provider.addBlock(BLOOD_ASPECT_LEAVES, "Blood Leaves");
-		provider.addBlock(BLOOD_ASPECT_SAPLING, "Blood Sapling");
-		provider.addBlock(BLOOD_ASPECT_PLANKS, "Blood Planks");
-		provider.addBlock(BLOOD_ASPECT_CARVED_PLANKS, "Carved Blood Planks");
-		provider.addBlock(BLOOD_ASPECT_STAIRS, "Blood Stairs");
-		provider.addBlock(BLOOD_ASPECT_SLAB, "Blood Slab");
-		provider.addBlock(BLOOD_ASPECT_FENCE, "Blood Fence");
-		provider.addBlock(BLOOD_ASPECT_FENCE_GATE, "Blood Fence Gate");
-		provider.addBlock(BLOOD_ASPECT_DOOR, "Blood Door");
-		provider.addBlock(BLOOD_ASPECT_TRAPDOOR, "Blood Trapdoor");
-		provider.addBlock(BLOOD_ASPECT_PRESSURE_PLATE, "Blood Pressure Plate");
-		provider.addBlock(BLOOD_ASPECT_BUTTON, "Blood Button");
-		provider.addBlock(BLOOD_ASPECT_BOOKSHELF, "Blood Bookshelf");
-		provider.addBlock(BLOOD_ASPECT_LADDER, "Blood Ladder");
+		provider.addBlock(BLOOD_ASPECT_LOG.blockHolder(), "Blood Log");
```

Comment:
`MinestuckLanguageProvider` has an overload for ItemBlockPair. You can therefore write this as `provider.add(BLOOD_ASPECT_LOG, "Blood Log");` (you can see this used in `SkaiaBlocksData`)